ecs-configuration/logstash/pipelines/suricata/logstash-601-filter-suricata-fileinfo.conf

```diff
@@ -9,7 +8,9 @@ filter {
           "[fileinfo][sha512]" => "[file][hash][sha512]"
           "[fileinfo][filename]" => "[file][name]"
           "[fileinfo][size]" => "[file][size]"
-        }
+          "[fileinfo][tx_id]" => "[file][inode]"
```

Comment:
In Suricata, `tx_id` represents the transaction ID so that you can associate this file with a network connection. Instead of `file.inode`, let's add this to `related.id` and maybe `event.id`.